test/e2e/play_kube_test.go

```diff
@@ -29,6 +29,37 @@ import (
 	"github.com/opencontainers/selinux/go-selinux"
 )
 
+var secretYaml = `
+apiVersion: v1
+kind: Secret
+metadata:
+  name: newsecret
+type: Opaque
+data:
+  username: dXNlcg==
+  password: NTRmNDFkMTJlOGZh
```

Comment:
I'd like an additional, more complex YAML with multiple secrets to make sure that mounting works for all secrets.